docs/technical_reference/unit_models/electrodialysis_0D.rst

```diff
@@ -0,0 +1,219 @@
+of water. The following key assumptions are based on.
+
+* The concentrate and diluate channels have identical geometry and fluidic conditions.
+* For each channel, component fluxes are uniform in the bulk solutions (the 0-dimensional assumption)
+  and are set as the average of inlet and outlet of each channel.
+* Steady state: all variables are independent on time.
+* Co-current flow operation. 
+* Electrical current is operated below the limiting current. 
+* Ideality assumptions: activity, osmotic, and van't Hoff coefficients are set at one. 
+* All ion-exchange membrane properties (ion and water transport number, resistance, permeability) are
+  constant.
+* Detailed concentration gradient effect at membrane-water interfaces is neglected. 
+* Constant pressure and temperature through each channel. 
+
+Control Volumes
+---------------
+
+This model has two control volumes for the concentrate and diluate channels.
+
+* Diluate_channel
+* Concentrate_channel
+
+Ports
+-----
+
+On the two control volumes, this model provides four ports (Pyomo notation in parenthesis):
+
+* inlet_diluate (inlet)
+* outlet_diluate (outlet)
+* inlet_concentrate (inlet)
+* outlet_concentrate (outlet)
+
+Sets
+----
+**Hereafter, a NaCl water solution is instanced to demonstrate the present model's use.**
+
+.. csv-table:: **Table 1.** List of Set
+   :header: "Description", "Symbol", "Indices"
+
+
+   "Time", ":math:`t`", "[0]"
+   "Phase", ":math:`p`", "['Liq']"
+   "Component", ":math:`j`", "['H2O', 'Na_+', '\Cl_-'] \ :sup:`1`"
+   "Ion", ":math:`j`\  :sup:`2`", "['Na_+', '\Cl_-']"
+   "Membrane", "n/a", "['cem', 'aem']"
+
+**Notes**
+ :sup:`1` A component set for a NaCl solution is instanced here.
+ :sup:`2` "Ion" is a subset of "Component" and uses the same symbol.
+
+
+Degrees of Freedom
+------------------
+Applying this model to a NaCl solution yields 33 degrees of freedom (**Table 2**), among which
+temperature, pressure, and component molar flow rate are state variables that are fixed as initial conditions. The rest
+are parameters that should be provided in order to fully solve the model.
+
+.. csv-table:: **Table 2.** List of Degree of Freedom (DOF)
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units", "DOF Number \ :sup:`1`"
+
+   "Temperature, inlet_diluate", ":math:`T`", "temperature", "None", ":math:`K`", 1
+   "Temperature, inlet_concentrate", ":math:`T`", "temperature", "None", ":math:`K`", 1
+   "Pressure, inlet_diluate",":math:`p`", "temperature", "None", ":math:`Pa`", 1
+   "Pressure, inlet_concentrate",":math:`p`", "temperature", "None", ":math:`Pa`", 1
+   "Component molar flow rate, inlet_diluate", ":math:`N`", "flow_mol_phase_comp", "[[t], ['Liq'],['H2O', 'Na_+', '\Cl_-']", ":math:`mol s^{-1}`", 3
+   "Component molar flow rate, inlet_concentrate", ":math:`N`", "flow_mol_phase_comp", "[[t], ['Liq'],['H2O', 'Na_+', '\Cl_-']", ":math:`mol s^{-1}`", 3
```

Comment:
I would prefer to keep `['H2O', 'Na_+', '\Cl_-']` to show this is indexed by all components including H2O , the line with the other one is indexed by ions.  This would not be reflected if using j.  I modified the doubled brackets for simplification.